### ISSUE TITLE
Webhook for multiple organisations

### ISSUE THREAD

**Issue by gitstashgithub** [calcinai/xero-php#824]
Hi guys,

I have a question about webhook. If our app is authorised for multiple Xero organisations, how do we make webhook work?
Below is the Webhook example from the document, you can see the first parameter for Webhook is XeroPHP\Application. 
```
// Configure the webhook signing key on the application
$application->setConfig(['webhook' => ['signing_key' => 'xyz123']]);
$webhook = new Webhook($application, $request->getContent());

/**
 * @return int
 */
$webhook->getFirstEventSequence();

/**
 * @return int
 */
$webhook->getLastEventSequence();

/**
 * @return \XeroPHP\Webhook\Event[]
 */
$webhook->getEvents();
```
However, to initialise the XeroPHP\Application variable, we need $tenantId.
```
$application = new \XeroPHP\Application($accessToken, $tenantId);
```

The problem is, there are multiple tenants(organisations), so we don't know the $tenantId. 

Does anyone know how to make it work? Thanks!

Gordon

**Comment by rodjsta** [comment]
The JSON Payload contains the tenantId
![image](https://user-images.githubusercontent.com/32949856/134269696-cd9d1d35-dd87-4cd9-b648-23f21270170a.png)

getEvents() will return you an array of Event classes. Each Event will have the tenantId and other information as per the invoming JSON.

**Comment by gitstashgithub** [comment]
@rodjsta thanks, but the problem is before this stage. 
You need `$webhook` to run `getEvents()`. 
To create `$webhook`, you need a `XeroPHP\Application` instance.
To initialise a `XeroPHP\Application` instance, you need `tenantId`.

**Comment by rodjsta** [comment]
@gitstashgithub, right... I see. Sorry, I actually handle my webhook events independent of this package. I essentially capture each event from the payload and process them separately.

[https://medium.com/swlh/handling-xero-webhooks-with-a-laravel-application-61eefd4492c4](https://medium.com/swlh/handling-xero-webhooks-with-a-laravel-application-61eefd4492c4)
